extension/src/popup/views/UnlockAccount/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import get from "lodash/get";
+import { useDispatch, useSelector } from "react-redux";
+import { useLocation } from "react-router-dom";
+import { Field, Formik, FieldProps } from "formik";
+import { Input, Button, TextLink } from "@stellar/design-system";
+
+import { ROUTES } from "popup/constants/routes";
+import { navigateTo, openTab } from "popup/helpers/navigate";
+import { newTabHref } from "helpers/urls";
+import { Form, FormRow } from "popup/basics/Forms";
+import { SubviewWrapper } from "popup/basics/AccountSubview";
+import { Header } from "popup/components/Header";
+import {
+  confirmPassword,
+  authErrorSelector,
+} from "popup/ducks/accountServices";
+
+import "./styles.scss";
+
+export const UnlockAccount = () => {
+  const location = useLocation();
+  const from = get(location, "state.from.pathname", "") as ROUTES;
+  const queryParams = get(location, "search", "");
+  const destination = from || ROUTES.account;
+
+  const dispatch = useDispatch();
+  const authError = useSelector(authErrorSelector);
+
+  interface FormValues {
+    password: string;
+  }
+  const initialValues: FormValues = {
+    password: "",
+  };
+
+  const handleSubmit = async (values: FormValues) => {
+    const { password } = values;
+    await dispatch(confirmPassword(password));
+    navigateTo(destination, queryParams);
+  };
+
+  return (
+    <>
+      <Header />
+      <SubviewWrapper>
+        <div className="UnlockAccount__header">
+          A <strong>Stellar</strong> wallet for <br />
```

Comment:
Do we need a line break there? It's usually not great to use them with text because it can break "not nicely" on smaller screen sizes. Maybe we could set max-width for the text container to control the text break?

Reply:
good point `max-width` works better, changed in some other parts as well
https://github.com/stellar/freighter/pull/362/commits/8e188291b3212a16afafcf145a479ed41df242ec